package.json

```diff
@@ -72,5 +72,10 @@
     "webpack": "^4.44.2",
     "webpack-cli": "^4.8.0",
     "webpack-dev-server": "^3.11.1"
+  },
+  "jest": {
```

Comment:
are the jest tests just run by this script?
```
    "test": "react-scripts test",
```
if not, then can you modify this or add a script to run them

Reply:
You can run npm test.  There are instructions in the read me.  I think they are the standard instructions generated by create react app.